grove/fermion_transforms/tests/test_jwtransform.py

```diff
@@ -0,0 +1,30 @@
+from grove.fermion_transforms.jwtransform import JWTransform
+
+
+def test_create():
+    """
+    Testing creation operator produces 0.5 * (X - iY)
+    """
+    jw = JWTransform()
+    creation = jw.create(5)
+    # we really need PauliSum and PauliTerm logical compare
```

Comment:
👍  on this I added an issue to pyquil for it:  https://github.com/rigetticomputing/pyquil/issues/41